src/components/record/filter/TemplateFilter.jsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { Button, Col, Form, Row } from "react-bootstrap";
+import { IntelligentTreeSelect } from "intelligent-tree-select";
+import { useI18n } from "../../../hooks/useI18n";
+import { sanitizeArray } from "../../../utils/Utils";
+import PropTypes from "prop-types";
+import { useDispatch, useSelector } from "react-redux";
+import { loadFormTemplates } from "../../../actions/FormTemplatesActions.js";
+
+const TemplateFilter = ({ value, onChange }) => {
+  const { i18n } = useI18n();
+  const dispatch = useDispatch();
+  const formTemplates = useSelector((state) => state.formTemplates.formTemplatesLoaded.formTemplates);
+  React.useEffect(() => {
+    if (!formTemplates) {
+      dispatch(loadFormTemplates());
+    }
+  }, [dispatch, formTemplates]);
+  const values = sanitizeArray(value);
+
+  const getFilterOptions = () => {
+    let templatesWithLabelAndValue = [];
+    for (const t of formTemplates) {
+      templatesWithLabelAndValue.push({
+        value: t["@id"],
+        label: t["http://www.w3.org/2000/01/rdf-schema#label"][0]["@value"],
```

Comment:
This is very crazy code ... JSON-LD might change and we will have problem, can you use something like:

JsonLdUtils.getJsonAttValue(item, Vocabulary.RDFS_LABEL)

Reply:
Will be addressed by https://github.com/kbss-cvut/record-manager-ui/issues/173.